tcl/pd_bindings.tcl

```diff
@@ -54,9 +54,16 @@ proc ::pd_bindings::global_bindings {} {
     # we use 'bind all' everywhere to get as much of Tk's automatic binding
     # behaviors as possible, things like not sending an event for 'O' when
     # 'Control-O' is pressed
+
+    if {$::windowingsystem eq "aqua"} {
+        set alt "Option"
+    } else {
+        set alt "Alt"
+    }
     bind_capslock all $::modifier-Key a {menu_send %W selectall}
     bind_capslock all $::modifier-Key b {menu_helpbrowser}
     bind_capslock all $::modifier-Key c {menu_send %W copy}
+    bind_capslock all $::modifier-$alt-Key c {menu_send %W copy-to-clipboard-as-text}
```

Comment:
the <kbd>Alt</kbd>/<kbd>Option</kbd> is *also* handled in `::pd_bindings::global_bindings`, i think  it should be handled only one (like with  `$::modifier`)